### ISSUE TITLE
Improved NER

### ISSUE THREAD

**Issue by JungeAlexander** [JungeAlexander/kbase_ner#1]
On 20210402 `spacy project run evaluate` yields the following results:

```
================================== Results ==================================

TOK      100.00
TAG      -
POS      -
MORPH    -
LEMMA    -
UAS      -
LAS      -
NER P    70.65
NER R    69.97
NER F    70.30
SENT P   -
SENT R   -
SENT F   -
SPEED    7830


=============================== NER (per type) ===============================

                  P       R       F
PRODUCT       68.25   66.93   67.58
PERSON        90.38   85.45   87.85
WORK_OF_ART    0.00    0.00    0.00
```

Things to try (probably in this order but TBD):

1. Label more data because:
    - Current dataset of ~100 documents was meant as initial attempt anyway 
    - Needs to be assisted by adding suggested labels based on rules/current models to save time during labeling, e.g. spacy EntityRuler, prodigy, snorkel
    - ~70 documents in first training set are too few
    - The fact that books (`WORK_OF_ART`), the smallest class, has an F of 0
2. Fine tuning embeddings on corpus
3. Introduce new labels instead of reusing `PRODUCT`, `PERSON`, and `WORK_OF_ART` 
4. Tweak model and hyperparameters
